ios-base/DroidKaigi 2020/Session/Views/Cells/SessionCell.swift

```diff
@@ -76,8 +76,30 @@ final class SessionCell: UICollectionViewCell {
         let view = UIControl()
         let speakerIconView = UIImageView()
         if let imageURL = imageURL {
-            let options = ImageLoadingOptions(transition: .fadeIn(duration: 0.3))
-            Nuke.loadImage(with: imageURL, options: options, into: speakerIconView)
+            // Configure cache
+            ImageCache.shared.costLimit = 1024 * 1024 * 50 // 50 MB
+            ImageCache.shared.countLimit = 100
+            ImageCache.shared.ttl = 120 // Invalidate image after 120 sec
+
+            ImageLoadingOptions.shared.transition = .fadeIn(duration: 0.3)
```

Comment:
These properties are `shared`. So I think we should set these values at one time. 🤔 
Please make common process about this? 🙏

Reply:
Thank you for the review!😄
`shared` is also used in [FloorMapViewController.swift](https://github.com/DroidKaigi/conference-app-2020/blob/196301157c6089e899a80bbe46d1f9add7eff9c6/ios-base/DroidKaigi%202020/FloorMap/FloorMapViewController.swift#L34-L39), so I will fix and commit again!